src/main/java/com/google/ehub/data/ProfileDatastore.java

```diff
@@ -22,6 +24,7 @@
   private static final String BIO_PROPERTY_KEY = "bio";
 
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  UserService userService = UserServiceFactory.getUserService();
```

Comment:
This field can be `private final` since its
- private: not used by other classes
- final: never reassigned